osu.Game.Modes.Osu/Objects/Drawables/Connections/HitObjectConnection.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics.Containers;
+using osu.Game.Modes.Objects.Drawables;
+using System.Collections.Generic;
+
+namespace osu.Game.Modes.Osu.Objects.Drawables.Connections
+{
+    public abstract class HitObjectConnection : Container
+    {
+        public abstract void AddConnections(IEnumerable<DrawableHitObject> drawableHitObjects, int startIndex = 0, int endIndex = -1);

```

Comment:
Can you also add an XML-doc comment explaining in detail what this method is supposed to do given the parameters? Just so implementers don't need to check the existing implementation and reverse-engineer it first.